addon/components/query-syntax/template.hbs

```diff
@@ -0,0 +1,100 @@
+        <p>
+            {{t 'eosf.components.querySyntax.use'}} <code>~</code> {{t 'eosf.components.querySyntax.fuzzinessDesc1'}}
+        </p>
+        <ul>
+            <li><code>girafe~ nekc~</code> -- {{t 'eosf.components.querySyntax.match'}} "giraffe neck".</li>
+        </ul>
+        <p>
+            {{t 'eosf.components.querySyntax.thisUsesThe'}} <a href='http://en.wikipedia.org/wiki/Damerau-Levenshtein_distance'>Damerau-Levenshtein distance</a> {{t 'eosf.components.querySyntax.damerauLevenshteinDistanceDesc'}} <code>~</code>.
+        </p>
+        <ul>
+            <li><code>girafe~2</code> -- {{t 'eosf.components.querySyntax.match'}} "giraffe", "gyrate", "pirate", etc.</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.phraseProximity'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.phraseProximityDesc1'}}
+        </p>
+        <ul>
+            <li><code>"giraffe neck"~2</code> -- {{t 'eosf.components.querySyntax.match'}} "giraffe has a neck", {{t 'eosf.components.querySyntax.butNot'}} "giraffe has a long neck".</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.ranges'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.rangesDesc1'}} <code>[min TO max]</code> {{t 'eosf.components.querySyntax.rangesDesc2'}} <code>{min TO max}</code> {{t 'eosf.components.querySyntax.rangesDesc3'}}.
+        </p>
+        <ul>
+            <li><code>date:[2016-01-01 TO 2016-12-31]</code> -- {{t 'eosf.components.querySyntax.match'}} {{t 'eosf.components.querySyntax.allDatesIn'}} 2016.</li>
+            <li><code>tags:{alpha TO omega}</code> -- {{t 'eosf.components.querySyntax.match'}} {{t 'eosf.components.querySyntax.allTagsBetween'}} "alpha" {{t 'eosf.components.querySyntax.and'}} "omega", {{t 'eosf.components.querySyntax.excluding'}} "alpha" {{t 'eosf.components.querySyntax.and'}} "omega".</li>
+            <li><code>date:[2016-01-01 TO 2016-12-31}</code> -- {{t 'eosf.components.querySyntax.match'}} {{t 'eosf.components.querySyntax.allDatesIn'}} 2016 {{t 'eosf.components.querySyntax.except'}} December 31.</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.boosting'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.boostingDesc1'}} <code>^</code> {{t 'eosf.components.querySyntax.boostingDesc2'}}
+        </p>
+        <ul>
+            <li><code>giraffe^2 lion</code> -- {{t 'eosf.components.querySyntax.boostingDesc3'}} "giraffe" {{t 'eosf.components.querySyntax.booleanDesc4'}} "lion".</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.moreInformation'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.moreInfoQuerySyntax'}} <a href='https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax'>Elasticsearch documentation</a>.
```

Comment:
I can translate documentation, but I don't think we should translate Elasticsearch.